centos-ci/run-treecompose

```diff
@@ -10,9 +11,12 @@ done
 
 # Update release tags
 ~/ostree-releng-scripts/do-release-tags --autocreate --repo=ostree/repo --releases=${buildscriptsdir}/releases.yml
-# Now, ensure we have a delta from N-1 -> N
-if ostree --repo=ostree/repo rev-parse centos-atomic-host/7/x86_64/devel/alpha^ 2>/dev/null; then
-    ostree --repo=ostree/repo static-delta generate -n centos-atomic-host/7/x86_64/devel/alpha
+smoketested_commit=$(ostree --repo=ostree/repo rev-parse ${refprefix}/smoketested)
+# Reset alpha to the latest smoketested; see https://github.com/CentOS/sig-atomic-buildscripts/issues/269
+~/ostree-releng-scripts/write-ref --ref ${refprefix}/alpha ${smoketested_commit}
+# Generate a delta for smoketested
+if ostree --repo=ostree/repo rev-parse ${refprefix}/smoketested^ 2>/dev/null; then
+    ostree --repo=ostree/repo static-delta generate -n ${refprefix}/smoketested
```

Comment:
Should we even bother generating static-deltas, given that it moves so fast? I guess it can still have value for testing purposes.